OPHD/ophd.vcxproj

```diff
@@ -106,9 +106,16 @@
       <AdditionalLibraryDirectories>$(SolutionDir)$(Configuration)\;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
     </Link>
     <PostBuildEvent>
-      <Command>SET sourceDir="$(VcpkgCurrentInstalledDir)bin\"
-if $(ConfigurationName) == Debug (
-  SET sourceDir="$(VcpkgCurrentInstalledDir)debug\bin\"
+      <Command>SET vcpkgDir="$(VcpkgCurrentInstalledDir)"
+
+IF "vcpkgDir" == "" (
+  EXIT
+)
+
+SET sourceDir="%vcpkgDir%bin\"
+
+IF $(ConfigurationName) == Debug (
+  SET sourceDir="%vcpkgDir%debug\bin\"

```

Comment:
I had to read this a couple times to understand the logic. And I'm maybe mildly concerned if `EXIT` works in this context.

What if we check the original variable first, and then set variables based on configuration.
```
IF NOT "$(VcpkgCurrentInstalledDir)" == "" (
  IF "$(ConfigurationName)" == "Debug" (
    SET configDir="debug\"
  ) ELSE (
    SET configDir=""
  )

  SET sourceDir="$(VcpkgCurrentInstalledDir)%configDir%bin\"

  xcopy /y /d "%sourceDir%ogg.dll" "$(TargetDir)"
  xcopy /y /d "%sourceDir%vorbis.dll" "$(TargetDir)"
  xcopy /y /d "%sourceDir%vorbisfile.dll" "$(TargetDir)"
)
```

The `ELSE` block might not be strictly necessary, though probably a good idea in case the variable was already defined in the environment.

Reply:
I thought you were asking me to write the script. :) No issues with the rewrite.

One thing I found concerning is when googling VcpkgCurrentInstalledDir, no documentation seems to come up. Maybe because it is a new feature? I would have expected mention in MSDN or maybe on the vcpkg github page.

I installed vcpkg I think 2-3 months ago, maybe a bit earlier. Agree that if VcpkgConfigSubdir is actually that new, probably better not to support for a while? If you want closure, I can reinstall and check. Otherwise I was going to move on for now.

Reply:
Haha, I know. I felt kind of funny after posting that. Thing is, I didn't really know what the form of the solution would be until you first posted something, and I'm not really able to test changes from Linux.

The `VcpkgCurrentInstalledDir` would be Vcpkg specific, so probably no documentation on MSDN. You'd need to check the Vcpkg documentation for that, assuming they even document it.

Vcpkg is under fairly active development, so probably quite a few changes in the last 2-3 months. At any rate, we don't need to be relying on bleeding edge features, so should be fine to just move on.

Reply:
I had to set delayed expansion to get to work. Otherwise, seems to be working on my machine. Please merge if CI builds are acceptable.

I thought vcpkg was actively developed by Microsoft.